tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -29,6 +29,41 @@ function print_usage() {
 	echo "Usage: $0 [install/cleanup/reset]"
 }
 
+function create_runtimeclasses() {
+	echo "Creating the runtime classes"
+
+	for shim in "${shims[@]}"; do
+		echo "Creating the kata-${shim} runtime class"
+		kubectl apply -f /opt/kata-artifacts/runtimeclasses/kata-${shim}.yaml
+	done
+
+	if [[ "${CREATE_DEFAULT_RUNTIMECLASS:-"false"}" == "true" ]]; then
+		echo "Creating the kata runtime class for the default shim (an alias for kata-${default_shim})"
+		cp /opt/kata-artifacts/runtimeclasses/kata-${default_shim}.yaml /tmp/kata.yaml
+		sed -i -e 's/${default_shim}/kata/g' /tmp/kata.yaml
+		kubectl apply -f /tmp/kata.yaml
+		rm -f /tmp/kata.yaml
+	fi
+}
+
+function delete_runtimeclasses() {
+	echo "Deleting the runtime classes"
+
+	for shim in "${shims[@]}"; do
+		echo "Deleting the kata-${shim} runtime class"
+		kubectl delete -f /opt/kata-artifacts/runtimeclasses/kata-${shim}.yaml
+	done
+
+
+	if [[ "${CREATE_DEFAULT_RUNTIMECLASS:-"false"}" == "true" ]]; then
+		echo "Deleting the kata runtime class for the default shim (an alias for kata-${default_shim})"
+		cp /opt/kata-artifacts/runtimeclasses/kata-${default_shim}.yaml /tmp/kata.yaml
+		sed -i -e 's/${default_shim}/kata/g' /tmp/kata.yaml
```

Comment:
This sed doesn't work.

```
[wmoschet@wainer-laptop kata-deploy]$ default_shim="qemu"
[wmoschet@wainer-laptop kata-deploy]$ cp ./runtimeclasses/kata-${default_shim}.yaml /tmp/kata.yaml
[wmoschet@wainer-laptop kata-deploy]$ sed -i -e 's/${default_shim}/kata/g' /tmp/kata.yaml
[wmoschet@wainer-laptop kata-deploy]$ cat /tmp/kata.yaml
---
kind: RuntimeClass
apiVersion: node.k8s.io/v1
metadata:
    name: kata-qemu
handler: kata-qemu
overhead:
    podFixed:
        memory: "160Mi"
        cpu: "250m"
scheduling:
  nodeSelector:
    katacontainers.io/kata-runtime: "true"
```
Now suppose:
```
[wmoschet@wainer-laptop kata-deploy]$ sed -i -e 's/'${default_shim}'/kata/g' /tmp/kata.yaml
[wmoschet@wainer-laptop kata-deploy]$ cat /tmp/kata.yaml
---
kind: RuntimeClass
apiVersion: node.k8s.io/v1
metadata:
    name: kata-kata
handler: kata-kata
overhead:
    podFixed:
        memory: "160Mi"
        cpu: "250m"
scheduling:
  nodeSelector:
    katacontainers.io/kata-runtime: "true"
```

Which is still wrong. I think you want:

```
[wmoschet@wainer-laptop kata-deploy]$ cp ./runtimeclasses/kata-${default_shim}.yaml /tmp/kata.yaml
[wmoschet@wainer-laptop kata-deploy]$ sed -i -e 's/kata-'${default_shim}'/kata/g' /tmp/kata.yaml
[wmoschet@wainer-laptop kata-deploy]$ cat /tmp/kata.yaml 
---
kind: RuntimeClass
apiVersion: node.k8s.io/v1
metadata:
    name: kata
handler: kata
overhead:
    podFixed:
        memory: "160Mi"
        cpu: "250m"
scheduling:
  nodeSelector:
    katacontainers.io/kata-runtime: "true"
```

Reply:
btw, I didn't understand why delete the "kata" runtimeclass.

Reply:
never mind to the previous question, I thought I was reading the `create_runtimeclasses()`

Reply:
You're correct!